### DISCUSSION TITLE
How to execute python program via popclip

### DISCUSSION THREAD

**Discussion by mafulong** [pilotmoon/PopClip-Extensions#1204] [General]
The ideal effect is that I can select a word text, and transfer it to my python3 program which can be called in my system environment. 
If it's not possible, my python program can also use the content from system clipborad, so I need poplicp to execute my python program. How to implement that? 
Thanks  a lot.

**Comment by pilotmoon** [comment]
You can include as python script in the extension executable. What the script does, is of course up to you. This page https://github.com/pilotmoon/PopClip-Extensions details how to make extensions. You need to look at the section about Shell Script Extensions in particular. Also you might find examples in the source and source-contrib directories.
